omnisafe/algorithms/model_based/base/ensemble.py

```diff
@@ -160,7 +160,9 @@ def __init__(
     def forward(self, input_data: torch.Tensor) -> torch.Tensor:
         """Forward pass."""
         w_times_x = torch.bmm(input_data, self.weight)
-        return torch.add(w_times_x, self.bias[:, None, :])  # w times x + b
+        if self.bias is not None:
+            return torch.add(w_times_x, self.bias[:, None, :])  # w times x + b
+        return w_times_x  # type: ignore
```

Comment:
The original code exists some error.
If `self.bias` is `None`, then `return torch.add(w_times_x, self.bias[:, None, :])` will take a slice of `None`. So I use an `if else` here.
Then, as `mypy` thinks that the code will not be reached, I wonder line 165 may be used in future model-based code, so I just disable `mypy` here.